types/culori/tsconfig.json

```diff
@@ -13,6 +13,9 @@
         "forceConsistentCasingInFileNames": true
     },
     "files": [
+        "all/index.d.ts",
+        "css/index.d.ts",
+        "fn/index.d.ts",
```

Comment:
I’m actually not sure if this change is needed; can revert if so (it seems like `src/` is missing)